[rocWMMA] 7.13 changelog updates (#6894)
ROCm/rocm-libraries: projects/rocwmma/CHANGELOG.md

Documentation for rocWMMA is available at
[https://rocm.docs.amd.com/projects/rocWMMA/en/latest](https://rocm.docs.amd.com/projects/rocWMMA/en/latest).

> ## rocWMMA 2.2.0 for ROCm 7.13.0

Reviewer: ```suggestion
## Since last release (ROCm 7.12)
```
We will not be using "component version a.b.c for rocm d.e.f" anymore for ROCm 7.13

### Added

> * Added community samples area for external contributions, with build support and documentation

Reviewer: ```suggestion
* Added the following community samples for external contributions, with build support and documentation
```

>   * Added community sample `simple_gemm_silu`: demonstrates GEMM + SiLU fused operator using the rocWMMA API

Reviewer: ```suggestion
  * `simple_gemm_silu`: demonstrates a GEMM + SiLU fused operator using the rocWMMA API
```

>   * Added community sample `simple_gemm_fusion`: demonstrates block-tile-level dual-GEMM fusion using the rocWMMA API

Reviewer: ```suggestion
  * `simple_gemm_fusion`: demonstrates block-tile-level dual-GEMM fusion using the rocWMMA API
```

>   * Added community sample `simple_gemm_swiglu`: demonstrates SwiGLU fused dual-GEMM kernel (LLaMA/Mistral FFN gate layer) using the rocWMMA API

Reviewer: ```suggestion
  * `simple_gemm_swiglu`: demonstrates a SwiGLU fused dual-GEMM kernel (LLaMA/Mistral FFN gate layer) using the rocWMMA API
```

### Changed

> * Updated `find_package` search for OpenMP to prefer the openmp-config.cmake shipped with ROCm, with fallback to module search mode

Reviewer: ```suggestion
* Updated the `find_package` search for OpenMP to prefer the `openmp-config.cmake` provided by ROCm, with a fallback to module search mode.
```
